gemrb/core/GameScript/Actions.cpp

```diff
@@ -6814,8 +6814,8 @@ void GameScript::UseItem(Scriptable* Sender, Action* parameters)
 		return;
 	}
 
-	act->UseItem(Slot, header, tar, flags);
 	Sender->ReleaseCurrentAction();
+	act->UseItem(Slot, header, tar, flags);
```

Comment:
This change should be applied to GameScript::UseItemPoint as well and ideally this PR would be in 3 commits, for the 3 different things it does.